Integrations/AzureSecurityCenter/AzureSecurityCenter.py

```diff
@@ -0,0 +1,1253 @@
+            requests_table_output.append({
+                'VirtualMachines': ', '.join(vms),
+                'Requestor': requestData.get('requestor') if requestData.get('requestor') else 'service-account',
+                'StartTimeUtc': requestData.get('startTimeUtc')
+            })
+        md = tableToMarkdown(
+            'Azure Security Center - Get JIT Access Policy - Requests',
+            requests_table_output,
+            [
+                'VirtualMachines',
+                'Requestor',
+                'StartTimeUtc'
+            ],
+            removeNull=True
+        )
+
+        requests_table_entry = {
+            'Type': entryTypes['note'],
+            'Contents': properties.get('requests'),
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': md,
+            'EntryContext': {}
+        }
+        demisto.results([property_table_entry, rules_table_entry, requests_table_entry])
+
+
+def get_jit(policy_name, asc_location, resource_group_name):
+    cmd_url = '/resourceGroups/{}/providers/Microsoft.Security/locations/{}/jitNetworkAccessPolicies/{}?api-version={}'.format(
+        resource_group_name, asc_location, policy_name, JIT_API_VERSION)
+    response = http_request('GET', cmd_url)
+    return response
+
+
+def initiate_jit_command(args):
+    resource_group_name = args.get('resource_group_name')
+    asc_location = args.get('asc_location')
+    policy_name = args.get('policy_name')
+    vm_id = args.get('vmID')
+    port = args.get('port')
+    source_address = args.get('source_address')
+    duration = args.get('duration')
+    response = initiate_jit(resource_group_name, asc_location, policy_name, vm_id, port, source_address, duration)
+    policy_id = "/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Security/locations/{}/jitNetworkAccessPolicies/{}".format(
+        SUBSCRIPTION_ID, resource_group_name, asc_location, policy_name)
+    virtual_machines = response.get('virtualMachines')
+    if virtual_machines and len(virtual_machines) > 0:
+        machine = virtual_machines[0]
+        port = machine.get('ports')[0]
+
+        outputs = {
+            'VmID': machine.get('id'),
+            'PortNum': port.get('number'),
+            'AllowedSourceAddress': port.get('allowedSourceAddressPrefix'),
+            'EndTimeUtc': port.get('endTimeUtc'),
+            'Status': port.get('status'),
+            'Requestor': response.get('requestor'),
+            'PolicyID': policy_id
+        }
+
+        md = tableToMarkdown(
+            'Azure Security Center - Initiate JIT Access Request',
+            outputs,
+            [
+                'VmID',
+                'PortNum',
+                'AllowedSourceAddress',
+                'EndTimeUtc',
+                'Status',
+                'Requestor'
+            ],
+            removeNull=True
+        )
+
+        demisto.results({
```

Comment:
use return_outputs instead of demisto.results

Reply:
will fix in the next phase (approved by anar)